aws/opsworks_layers.go

```diff
@@ -214,6 +222,101 @@ func (lt *opsworksLayerType) SchemaResource() *schema.Resource {
 			Computed: true,
 		},
 		"tags": tagsSchema(),
+		"cloudwatch_configuration": {
+			Type:     schema.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+				if old == "1" && new == "0" && !d.Get("cloudwatch_configuration.0.enabled").(bool) {
+					return true
+				}
+				return false
+			},
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"enabled": {
+						Type:     schema.TypeBool,
+						Optional: true,
+						DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+							if old == "false" && new == "" {
+								return true
+							}
+							return false
+						},
+					},
+					"log_streams": {
+						Type:     schema.TypeList,
+						Optional: true,
+						DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+							if old == "1" && new == "0" && !d.Get("cloudwatch_configuration.0.enabled").(bool) {
+								return true
+							}
+							return false
+						},
+						Elem: &schema.Resource{
+							Schema: map[string]*schema.Schema{
+								"batch_count": {
+									Type:         schema.TypeInt,
+									Default:      1000,
+									Optional:     true,
+									ValidateFunc: validation.IntAtMost(10000),
+								},
+								"batch_size": {
+									Type:         schema.TypeInt,
+									Default:      32768,
+									Optional:     true,
+									ValidateFunc: validation.IntAtMost(1048576),
+								},
+								"buffer_duration": {
+									Type:         schema.TypeInt,
+									Default:      5000,
+									Optional:     true,
+									ValidateFunc: validation.IntAtLeast(5000),
+								},
+								"datetime_format": {
+									Type:     schema.TypeString,
+									Optional: true,
+								},
+								"encoding": {
+									Type:         schema.TypeString,
+									Optional:     true,
+									Default:      opsworks.CloudWatchLogsEncodingUtf8,
+									ValidateFunc: validation.StringInSlice(opsworks.CloudWatchLogsEncoding_Values(), false),
+								},
+								"file": {
+									Type:     schema.TypeString,
+									Required: true,
+								},
+								"file_fingerprint_lines": {
+									Type:     schema.TypeString,
+									Optional: true,
+									Default:  "1",
+								},
+								"initial_position": {
+									Type:         schema.TypeString,
+									Optional:     true,
+									Default:      opsworks.CloudWatchLogsInitialPositionStartOfFile,
+									ValidateFunc: validation.StringInSlice(opsworks.CloudWatchLogsInitialPosition_Values(), false),
+								},
+								"log_group_name": {
+									Type:     schema.TypeString,
+									Required: true,
+								},
+								"multiline_start_pattern": {
+									Type:     schema.TypeString,
+									Optional: true,
+								},
+								"timezone": {
```

Comment:
Strictly this should be `time_zone` as the API field is name `TimeZone`, as for `aws_mq_broker`, not `Timezone` as it is for `aws_db_instance`.

```suggestion
								"time_zone": {
```

Reply:
done